src/js/components/Index.js

```diff
@@ -255,7 +268,7 @@ Index.propTypes = {
   footer: PropTypes.node,
   inlineFilterParams: PropTypes.shape({
     onToggle: PropTypes.func,
-    defaultOpen: PropTypes.bool
```

Comment:
can we keep the defaultOpen for a while and remove it in the next major version?